etc/requirements_certifi.txt

```diff
@@ -0,0 +1 @@
+certifi==certifi=2018.8.24
```

Comment:
Is there a reason to install `certifi` separately, or could we move this into requirements.txt? I think it's not optional, while TA-lib is.